### PR TITLE
Updates openapi json schema generator

### PR DESCRIPTION
**Pull request by spacether**
**GitHub Issue:** # / NA

**Summary**:

Updates openapi json schema generator
- lists the project in java + python for validation
- lists the project in java and python for code generation, adds json schema drafts

I am updating these sections because I am the maintainer of that project and I [recently added draft 2020-12 schema validation to the java code generator](https://github.com/openapi-json-schema-tools/openapi-json-schema-generator/releases/tag/4.1.0).

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**

Yes/No
[JUSTIFICATION]

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -2,5 +2,16 @@
 <project version="4">
   <component name="VcsDirectoryMappings">
     <mapping directory="$PROJECT_DIR$" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/_includes/community" vcs="Git" />
```
Comment (Relequestual) [inline .idea/vcs.xml:5]: I'm not sure what this file is or why it has been modified in this PR, but I don't think it should be in this PR.

**Comment by Relequestual** [comment]
Thanks for your contribution. I've let one comment but otherwise this looks fine =]

File: .idea/vcs.xml (hunk 1 above)
Comment (spacether) [reply .idea/vcs.xml]: This is an intelij file
I did not manually edit it
Intelij/the readme's git commands updated this
Just now I reverted it

Comment (Relequestual) [reply .idea/vcs.xml]: Thanks for fixing.
Is the whole `.idea` directory something controlled by Intelij?
I think we should probably add it to the `.gitignore` file.
(Not in this PR, just in general. A new PR.)

Comment (spacether) [reply .idea/vcs.xml]: Yes it is

**Review by Relequestual — APPROVED** [review]

**Comment by spacether** [comment]
Thank you for the review and approval. How does merging work here? Will someone else do it later?

**Review by benjagm — APPROVED** [review]
Looks good to me. Thanks for the updates @spacether!

**Comment by benjagm** [comment]
Merged!

**Pull request merged**